Source/WebKit/Platform/IPC/Connection.h

```diff
@@ -264,9 +276,15 @@ class Connection : public ThreadSafeRefCountedAndCanMakeThreadSafeWeakPtr<Connec
             : handle(handle)
         {
         }
+        Identifier& operator=(Identifier&&) = default;
         operator bool() const { return !!handle; }
         HANDLE handle { 0 };
 #elif OS(DARWIN)
+        explicit Identifier(Identifier&& identifier)
```

Comment:
@Why can't we use `= default;` here?

Reply:
Mm, I bet we can, on all platforms. I'll give it a try tomorrow and see if EWS approves.

Reply:
Works on Linux. Let's try it on EWS.